eta-company-calendar-frontend/src/app/login/components/login.component.ts

```diff
@@ -0,0 +1,88 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { TranslateService } from '@ngx-translate/core';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { AuthService } from '~/app/shared/services/auth.service';
+
+@Component({
+  selector: 'app-welcome-description',
+  template: `
+  <form [formGroup]="loginForm" (ngSubmit)="onSubmit()">
+    <mat-form-field>
+      <mat-label>{{ 'login.email' | translate }}</mat-label>
+      <input matInput formControlName="email" type="text">
+    </mat-form-field>
```

Comment:
a form-fieldeken belül mehet mat-error mondjuk
`          <mat-error>
            {{'login.email-error' | translate}}
          </mat-error>`
ebben az esetben ha a formcontrol hibás kiirathatod vele hogy mi is a baj